### DISCUSSION TITLE
请问下，安装好后网页打不开是什么情况

### DISCUSSION THREAD

**Discussion by xifeng0** [AlistGo/alist#521] [Q&A]
使用的甲骨文云的实例安装，防火墙未开启，网络安全策略组也是把出入站所有IP和端口都放行了。但是就是503打不开网页。
![QQ图片20220211060918](https://user-images.githubusercontent.com/76558682/153505065-656529ce-795a-4341-a44e-0e6febf38f8b.png)
![QQ图片20220211060912](https://user-images.githubusercontent.com/76558682/153505099-2812df74-5d23-46d4-9b62-d430c8d5031e.png)
![QQ图片20220211060916](https://user-images.githubusercontent.com/76558682/153505126-d48d592b-54b0-4a86-baf2-34ba71026587.png)

**Comment by xhofe** [comment]
- #227
